recipe/bld.bat

```diff
@@ -1,23 +1,23 @@
-mkdir build
-if errorlevel 1 exit 1
+mkdir build && cd build
 
-cd build
-if errorlevel 1 exit 1
+set CMAKE_CONFIG=Release
 
-cmake .. ^
-        -G %CMAKE_GENERATOR% ^
-        -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
-        -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
-        -DBUILD_STATIC_LIBS=1 ^
-        -DBUILD_SHARED_LIBS=1 ^
-        -DPYTHON_EXECUTABLE="%PYTHON%"
-if errorlevel 1 exit 1
+rem shared libs fail to build, built static instead
+rem https://github.com/open-source-parsers/jsoncpp/issues/631
 
-cmake --build . --config Release
+cmake ^
+  -G "NMake Makefiles" ^
+  -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+  -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+  -DCMAKE_BUILD_TYPE="%CMAKE_CONFIG%" ^
+  -DBUILD_STATIC_LIBS=OFF ^
+  -DBUILD_SHARED_LIBS=ON ^
```

Comment:
Ouch! Not sure how we didn't run into that sooner. Thanks for correcting it. Do you want to copy that over as a patch now or are you wanting to wait for feedback on that PR?